src/interfaces/wallet.h

```diff
@@ -25,7 +24,10 @@ class CCoinControl;
 class CFeeRate;
 class CKey;
 class CWallet;
+enum isminetype : unsigned int;
 enum class FeeReason;
+typedef uint8_t isminefilter;
```

Comment:
nit: `using`?

Reply:
Yeah, but should change it in the original header file as well.